src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -153,7 +153,7 @@
                                                               :defaultValue description
                                                               :style {:width 400}
                                                               :rows 5}))
-                description description
+                description [:div {:style {:whiteSpace "pre"}} description]
```

Comment:
Back in the day, when I learned html in the early 90's, pre meant pre-formatted and preserved both white spaces and line wraps. I think that's still true today: http://www.w3schools.com/tags/tag_pre.asp